source/administration/plugins.rst

```diff
@@ -38,7 +52,7 @@ Once a plugin is installed (or pre-installed if shipped with Mattermost binary r
 Upgrading Plugins
 ~~~~~~~~~~~~~~~~~
 
-Upgrade a plugin on demand when a new version becomes available.
+Upgrade a plugin on demand when a new version becomes available.  New versions of plugins you have already installed will be shown and can be installed with a single click. 
```

Comment:
Pedantically, two clicks are required to confirm the upgrade.

Reply:
lol, true.